model_compression_toolkit/qat/pytorch/quantizer/quantization_dispatcher_builder.py

```diff
@@ -30,6 +32,42 @@
                                            qi.QuantizationMethod.UNIFORM: STEUniformActivationQuantizer}}
 
 
+def get_trainable_quantizer_weights_config(n: common.BaseNode):
```

Comment:
Isn't get_trainable_quantizer_weights_config (and activation as well) is similar to the one in Keras?
If so, why not put it in common and use the same method in both? dispachers?

Reply:
you're right it's common. but we decided this function to be under BaseNode, so this is a temporal solution till we remove dependency of QI in MCT. I will explain in a call...

Reply:
Ok, consider adding a TODO comment near those methods in both places